config/env.config.js

```diff
@@ -55,5 +55,46 @@ const config = {
     }
 };
 
+const RedirectPlugin = function(options) { 
+    this.apply = function(compiler) {
+        compiler.plugin('compilation', function(compilation) {
+            compilation.plugin('html-webpack-plugin-before-html-processing', function(htmlPluginData, callback) {
+                let htmlHead = htmlPluginData.html.match('<head>').index;
+                htmlPluginData.html = htmlPluginData.html.slice(0, htmlHead) + '<head>' +
+                    `<script>
+                        var getParameterByName = function(name) {
+                            var url = window.location.search;
+                            var queryExp = new RegExp("[\\?&]"+name+"=([^&#]*)", "i");
+                            var match = queryExp.exec(url);
+                            if (match && match.length > 1) {
+                                return match[1];
+                            }
+                            return "";
+                        };
+                        var originUrl = decodeURIComponent(getParameterByName("originEnvironment"));
+                        var newUrl = window.localStorage.getItem("environmentConfig");
+                        if (newUrl && !window.location.href.match(newUrl) && originUrl !== newUrl) {
```

Comment:
Can you run the approach to solve this issue by @Zlatkovsky and I before starting on the implementation.